index.js

```diff
@@ -20,6 +20,10 @@ module.exports = function (opts, cb) {
         if (typeof cb === 'function') cb(code, sig)
     });
 
+    ps.stdin.on('error', function (e) {
+        if (!e.errno || e.errno !== 'EPIPE') { throw(e); }
```

Comment:
Yeah, I wonder what's the best we could do here.

1. We can't re-emit an error event because we are not able to suppress it in the first place.
2. Yet handling error event is the only way to ignore EPIPE and ECONNRESET.
3. These errors are not in any sense fatal, they're just distraction.

So unless you come up with something better I suggest to just add an empty handler with a comment for future reference.

```js
ps.stdin.on('error', function () {
    // Ignore EPIPE and ECONNRESET.                                             
});
```